src/api/contributors/contributors.controller.js

```diff
@@ -5,7 +5,51 @@ module.exports = async function getContributors(req, res) {
   try {
     const response = await fetch(contribURL);
     const data = await response.json();
-    return res.json(data);
+    const finalResponse = {
+      data: data
+        .sort((a, b) => new Date(a.joined) - new Date(b.joined))
+        .map(({
+          name,
+          github: username,
+          image,
+          countryCode,
+          teamIds,
+          active,
+          joined,
+        }, key) => ({
+          type: 'contributor',
+          id: key,
+          attributes: {
+            username,
+            name,
+            image,
+            countryCode,
+            active,
+            joined,
+          },
+          relationships: {
+            contributionArea: {
+              links: {
+                self: `/contributors/${key}/relationships/contribution-areas`,
+                related: `/contributors/${key}/contribution-areas`,
+              },
+              data: teamIds.map(team => ({
+                type: 'contribution_area',
+                id: team,
+              })),
+            },
+          },
+        })),
+      included: [
+        { type: 'contribution-area', id: '0', attributes: { name: 'Planning' } },
+        { type: 'contribution-area', id: '1', attributes: { name: 'Design' } },
+        { type: 'contribution-area', id: '3', attributes: { name: 'Frontend' } },
+        { type: 'contribution-area', id: '4', attributes: { name: 'Backend' } },
+        { type: 'contribution-area', id: '2', attributes: { name: 'DevOps' } },
+        { type: 'contribution-area', id: '5', attributes: { name: 'Functional Testing' } },
+      ],
+    };
+    return res.json(finalResponse);
   } catch ({ message }) {
     return res.status(400).json({ status: 400, message });
```

Comment:
How is this a 400? Shouldn't it be 500 for unidentified server error? It seems to have nothing to do with user input.

Reply:
I realise this is not part of your code but since we are already working on this route we should also fix this.

Reply:
Ill make this change in the next commit.